app/src/index.tsx

```diff
@@ -2,16 +2,10 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
-import reportWebVitals from "./reportWebVitals";
```

Comment:
I'll stress again that we shouldn't be including extra changes in this already large PR, but we can leave this.

Please do not add any more extras that aren't strictly related to the feature we're attempting to implement.